src/sentry/static/sentry/app/views/performance/data.tsx

```diff
@@ -83,6 +83,23 @@ export function getFrontendAxisOptions(
   ];
 }
 
+export function getBackendAxisOptions(
+  organization: LightWeightOrganization
+): TooltipOption[] {
+  return [
+    {
+      tooltip: getTermHelp(organization, 'p75'),
+      value: `p75()`,
+      label: t('Duration p75'),
+    },
+    {
+      tooltip: getTermHelp(organization, ''),
```

Comment:
Just put anything in for now, we can fix it later. `distribution` -> `Distribution buckets counts of transactions at specifics times for your current date range`.